### DISCUSSION TITLE
Help Please! How can i make Time column in exported CSVs to show 24h format

### DISCUSSION THREAD

**Discussion by unudelaradio** [AzuraCast/AzuraCast#6873] [Q&A]
Hello,

I'm exporting Song Playback Timeline reports very often and i was wondering if there's a way to make the Time column to show 24h format. From 00:00 to 23:59.

This will help me a lot in reordering the full table by Time column. I tried to do this in Libre Office, changing Cell Format to HH:MM but without success. I guess the Time column values are imported as Text by Libre Office.

**Comment by Moonbase59** [comment]
Okayyyy… here’s a short "step-by-step" for LibreOffice:

1. In LibreOffice’s Import Dialog, set "separated" ("Getrennt") by "Comma" ("Komma"), then click the "Date" column header and choose the "Date (YMD)" ("Datum (JMT)") format.

![Textimport -  niteradio_timeline_2024-01-01_00-00-00_to_2024-01-18_23-59-59 csv _001](https://github.com/AzuraCast/AzuraCast/assets/3706922/719eb93b-2b3c-4b87-9527-a9a8b9fe8e4b)

2. Select the "Time" column and select "US English" ("US-Englisch"). Then click "Ok" to import.

![Textimport -  niteradio_timeline_2024-01-01_00-00-00_to_2024-01-18_23-59-59 csv _002](https://github.com/AzuraCast/AzuraCast/assets/3706922/9a2f38d4-cd36-432c-877b-e5a2bec83c06)

3. In the spreadsheet, select the "Time" column (it should show long decimals between 0 and 1). Then right-click in the marked column and choose "Format cells" ("Zellen formatieren").

![niteradio_timeline_2024-01-01_00-00-00_to_2024-01-18_23-59-59 csv - LibreOffice Calc_002](https://github.com/AzuraCast/AzuraCast/assets/3706922/d8c16c62-7ae1-4698-9826-74151c86ce33)

4. In the Format Cells dialog, select "Time" ("Uhrzeit") and the format you desire (I chose "HH:MM:SS"/"13:37:46" here), then click Ok to apply

![Zellen formatieren_001](https://github.com/AzuraCast/AzuraCast/assets/3706922/47986ddc-2d78-4b06-aa65-29f75a8384ec)

5. Et voilà! Dates formatted as `YYYY-MM-DD`, Times formatted as `HH:MM:SS`!

![niteradio_timeline_2024-01-01_00-00-00_to_2024-01-18_23-59-59 csv - LibreOffice Calc_001](https://github.com/AzuraCast/AzuraCast/assets/3706922/ff650eaa-d1eb-42b5-824d-4c000f1251fd)

Hope that helps!

**Reply by unudelaradio** [reply]
Wow! Many thanks for your patience in doing this tutorial! It works! 

After spending 3 hours to find out how to get the desired format, yesterday morning i figured out another way to reach this result. 

First, i formatted an empty column with the desired HH:MM:SS format. And after that, i copied the entire Time column and paste it as Unformatted Text in the new formatted-column. Using Paste Special (see screenshot).

Now i have 2 available ways! Awesome! Many thanks once again!

![Paste Special](https://github.com/AzuraCast/AzuraCast/assets/74244061/62c5e6a6-fbd5-4a7b-8511-383627cd8be7)

**Comment by Moonbase59** [comment]
![niteradio_timeline_2024-01-01_00-00-00_to_2024-01-18_23-59-59 csv - LibreOffice Calc_003](https://github.com/AzuraCast/AzuraCast/assets/3706922/4394292f-a90b-498b-8a23-25d754977760)

**Comment by Moonbase59** [comment]
@unudelaradio Happy it worked out for you! There are always a few little-known tricks (like "US English" to decode 12:48pm—who would think of _that_?) :-)
